modules/apps/object/object-service/src/main/java/com/liferay/object/internal/field/business/type/PicklistObjectFieldBusinessType.java

```diff
@@ -186,55 +186,45 @@ public void predefineObjectFieldSettings(
 	@Override
 	public void validateObjectFieldSettingsDefaultValue(
 			ObjectField objectField,
-			List<ObjectFieldSetting> objectFieldSettings)
+			Map<String, String> objectFieldSettingsValuesMap)
 		throws PortalException {
 
-		if (objectFieldSettings.isEmpty()) {
+		if (objectFieldSettingsValuesMap.isEmpty()) {
 			return;
 		}
 
-		ObjectFieldSetting defaultValueObjectFieldSetting = null;
+		ObjectFieldBusinessType.super.validateObjectFieldSettingsDefaultValue(
+			objectField, objectFieldSettingsValuesMap);
 
-		for (ObjectFieldSetting objectFieldSetting : objectFieldSettings) {
-			if (StringUtil.equals(
-					objectFieldSetting.getName(),
-					ObjectFieldSettingConstants.NAME_DEFAULT_VALUE)) {
-
-				defaultValueObjectFieldSetting = objectFieldSetting;
-			}
-
-			if (objectFieldSetting.compareName(
-					ObjectFieldSettingConstants.NAME_DEFAULT_VALUE_TYPE) &&
-				StringUtil.equals(
-					objectFieldSetting.getValue(),
-					ObjectFieldSettingConstants.VALUE_EXPRESSION_BUILDER)) {
+		String defaultValue = objectFieldSettingsValuesMap.get(
+			ObjectFieldSettingConstants.NAME_DEFAULT_VALUE);
+		String defaultValueType = objectFieldSettingsValuesMap.get(
+			ObjectFieldSettingConstants.NAME_DEFAULT_VALUE_TYPE);
 
-				return;
-			}
-		}
+		if ((defaultValue == null) ||
+			StringUtil.equals(
+				defaultValueType,
```

Comment:
Great job, @devpedrotavares @murilostodolni!

We can inline here, but anyway LGTM!